src/java.base/share/classes/java/lang/System.java

```diff
@@ -833,9 +833,9 @@ public static native void arraycopy(Object src,  int  srcPos,
      * {@link #clearProperty(String)} may not have the desired effect.
      *
      * @crac System properties can be updated on restore from a checkpoint.
-     * The application can {@link javax.crac.Context#register(javax.crac.Resource) register}
+     * The application can {@link jdk.internal.crac.mirror.Context#register(jdk.internal.crac.mirror.Resource) register}
```

Comment:
The docs should use the `jdk.crac` package; `jdk.internal.crac.mirror` should not be used from application. Could you check that the generated javadocs will then link correctly to the different module?

Reply:
No, just swapping `jdk.internal.crac.mirror` with `jdk.crac` here does not work: javadoc complains that it cannot find the reference. I am not sure how to create inter-module links, there is `{@link java.logging/java.util.logging.Logger java.util.logging.Logger}` syntax in this file which successfully links to another module but using `jdk.crac/jdk.crac` does not work.

The current `jdk.internal.crac.mirror` also creates no link, probably because no docs are generated for this package. There is no error, just no hyperlink in the resulting HTML.

Reply:
> The current jdk.internal.crac.mirror also creates no link, probably because no docs are generated for this package. 

It's expected that there's no docs for `jdk.internal.crac.mirror`; it should not be exposed, neither from code nor from docs. If you can't get links to `jdk.crac` work correctly you could fallback to just `{@code ...}`. The correctness of the information for users has priority over the form.

Reply:
Added a fix, turned out no docs were being built at all for the new module. After fixing that just changing the package to `jdk.crac` starting working but I used the more verbose `jdk.crac/jdk.crac.Something` syntax anyway to be inline with the rest of inter-module links in the file.

Reply:
Sadly, this fix is not enough. The docs themselves build fine but when compiling there are warnings that the referenced module cannot be found. Investigating.